src/core/lib/security/security_connector/security_connector.cc

```diff
@@ -959,24 +959,29 @@ static grpc_slice compute_default_pem_root_certs_once(void) {
 }
 
 static grpc_slice default_pem_root_certs;
+static ssl_root_certs_store* default_root_store = nullptr;
```

Comment:
Since this is static, I don't think that it's strictly necessary to destroy it.

Reply:
@markdroth wants to destroy it explicitly.